docs/release-notes/NuGet-5.5.md

```diff
@@ -0,0 +1,118 @@
+
+<sup>1</sup>Installed with Visual Studio 2019 with .NET Core workload
+
+## Summary: What's New in 5.5
+
+* Several accessibility fixes in PM UI
+
+* dotnet.exe reliability fixes with cross platform authentication plugins
+
+* msbuild /restore works with Packages.Config
+
+* Change the way we read XML files
+
+**Feature:**
+
+* add `dotnet nuget <add|remove|update|disable|enable|list> source` command - [#4126](https://github.com/NuGet/Home/issues/4126)
+
+* Improve network diagnostics to understand real world http download perf in VS - [#8592](https://github.com/NuGet/Home/issues/8592)
+
+* --skip-duplicate support for dotnet nuget push - [#8778](https://github.com/NuGet/Home/issues/8778)
+
+* msbuild /restore works with Packages.Config - [#8506](https://github.com/NuGet/Home/issues/8506)
+
+**DCR:**
+
+* nuget.exe pack should no longer warn about packing a SemVer 2.0.0 package - [#5201](https://github.com/NuGet/Home/issues/5201)
+
+* Embedded Icons support in NuGet PackageManager UI - [#8189](https://github.com/NuGet/Home/issues/8189)
+
+* Logic in _GetRestoreProjectStyle should be in a task - [#8804](https://github.com/NuGet/Home/issues/8804)
+
+* Deprecation info is not visible, by default, on installed tab - [#8541](https://github.com/NuGet/Home/issues/8541)
+
+**Bug:**
+
+* Rework Self-Updater with V3 Apis - [#4197](https://github.com/NuGet/Home/issues/4197)
+
+* Wrong package dependency version If package dependency version is set to '*' - [#6697](https://github.com/NuGet/Home/issues/6697)
+
+* ErrorUnsafePackageEntry error message is not pointing to source of problem - [#7505](https://github.com/NuGet/Home/issues/7505)
+
+* dotnet restore failing with TaskCanceledException - [#7842](https://github.com/NuGet/Home/issues/7842)
+
+* Lock file is not honored in "*" scenarios  - [#8073](https://github.com/NuGet/Home/issues/8073)
+
+* NuGet.exe does not resolve to the latest version of a package when using * in PackageReference (MSBuild/Dotnet/VS restore do) - [#8432](https://github.com/NuGet/Home/issues/8432)
+
+* dotnet list package with multi targeting WPF project - [#8463](https://github.com/NuGet/Home/issues/8463)
+
+* Plugin:  "A task was cancelled" - problem with ADO authentication due to this. - [#8528](https://github.com/NuGet/Home/issues/8528)
+
+* Improve ConcurrencyUtilities (reduce CPU usage) - [#8653](https://github.com/NuGet/Home/issues/8653)
+
+* DG Spec for unloaded project scenarios should not be written in preview restores - [#8793](https://github.com/NuGet/Home/issues/8793)
+
+* The Visual Studio NuGet packages (RestoreManagerPackage) needs to auto load on solution build events - [#8796](https://github.com/NuGet/Home/issues/8796)
+
+* [Test Failure] Package icon becomes a default icon after installing ‘Newtonsoft.Json’ with latest versoin - [#8814](https://github.com/NuGet/Home/issues/8814)
```

Comment:
Exclude this. 
We don't need to include the bugs that were regressed in the current release.

Reply:
mentioned about this feedback in NuGet/Client.Engineering/issues/228